documentation/schema.json

```diff
@@ -1582,6 +1593,43 @@
         "Triage"
       ]
     },
+    "CollectExceptionsOptions": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "Egress": {
+          "type": "string",
+          "description": "The name of the egress provider to which the exceptions are egressed.",
+          "minLength": 1
+        },
+        "Format": {
+          "description": "The format used to display the exceptions.",
+          "default": "PlainText",
+          "oneOf": [
+            {
+              "type": "null"
+            },
+            {
+              "$ref": "#/definitions/ExceptionsFormat"
+            }
+          ]
+        }
+      }
+    },
+    "ExceptionsFormat": {
```

Comment:
It looks like we have a mix of `ExceptionsFormat` and `ExceptionFormat` - based on the other changes here, I'm assuming we'd want to standardize on the singular `ExceptionFormat`?